en/docs/get-started/configure-users-and-roles.md

```diff
@@ -66,4 +66,79 @@ Now you have started the servers, let’s create the users and define their perm
     ![set_new_user_roles](../assets/img/get-started/quick-start-guide/set-new-user-roles.png)
         
     iii. Click **Finish**.
+
+## Configuring SMS OTP Authenticator
+
+1. Sign in to the [Management Console](https://localhost:9446/carbon) as an administrator
+
+2. Adding SMS OTP Identity Provider:
+
+     - Navigate to the Main menu to access the Identity menu and select **Identity Providers** -> **Add**.
+     
+        ![add_identity_providers](../assets/img/get-started/quick-start-guide/go-to-add-identity-providers.png)
+
+     - Fill the **Basic Information** section and name this identity provider as **SMSAuthentication**
+
+        ![sms_authentication_idp](../assets/img/get-started/quick-start-guide/sms-authentication-idp.png)
+
+     - Expand the **Federated Authenticators** -> **SMS OTP Configuration** section
+
+        ![expand_federated_authenticators](../assets/img/get-started/quick-start-guide/expand-federated-authenticators.png)
+
+     - Select both the **Enable** and **Default** checkboxes. This is to enable and make the SMSAuthentication authenticator the default one
+
+        ![config_sms_otp](../assets/img/get-started/quick-start-guide/config-sms-otp.png)
+
+     - Add the following sample configurations and click **Register**
+
+        ```
+         SMS URL : https://api.twilio.com/2010-04-01/Accounts/AC34f40df03e20fb6498b3fcee256ebd3b/SMS/Messages.json
+         HTTP Headers : Authorization: Basic QUMzNGY0MGRmMDNlMjBmYjY0OThiM2ZjZWUyNTZlYmQzYjo1ZmFkM2VkYzg4YWM1NTNiMmFiZjc4 NWI1MmM4MWFkYg==
+         HTTP Payloads : Body=$ctx.msg&To=$ctx.num&From=+1 210-880-1806
+         HTTP Method : POST
+        ```
+       
+3. Configuring Account Lock:
+
+    - Navigate to the Identity menu and select **Identity Providers** -> **Resident** -> **Login Attempt Security** -> **Account Lock**
+
+        ![select_resident_identity](../assets/img/get-started/quick-start-guide/go-to-resident-identity-providers.png)
+
+       ![security_account_lock](../assets/img/get-started/quick-start-guide/login-security-account-lock.png)
+    
+    - Select the **Lock User Accounts** checkbox
+
+       ![maximum_login_attempts](../assets/img/get-started/quick-start-guide/maximum-failed-login-attempts.png)
+
+    - Configure the **Maximum Failed Login Attempts** (default 5)
```

Comment:
can we explain what "(default 5)" is?

Reply:
By default the maximum failed login attempts are 5 but you can change it